src/services/action/utils/utils.ts

```diff
@@ -0,0 +1,73 @@
+import { AggregateBy, AggregateFunction, AggregateMetric, CountGroupBy } from '@graasp/sdk';
+
+import {
+  AggregateByCannotIncludeAggregateMetricError,
+  AggregateByCannotUserError,
+  CountGroupByShouldIncludeAggregateByError,
+  CountGroupByShouldIncludeAggregateMetricError,
+  InvalidAggregateFunctionError,
+} from './errors';
+
+export const validateAggregationParameters = ({
+  countGroupBy,
+  aggregationParams,
+}: {
+  countGroupBy?: CountGroupBy[];
+  aggregationParams?: {
+    aggregateFunction?: AggregateFunction;
+    aggregateMetric?: AggregateMetric;
+    aggregateBy?: AggregateBy[];
+  };
+}) => {
+  const { aggregateFunction, aggregateMetric, aggregateBy } = aggregationParams ?? {};
+
+  // Aggregate by user is not allowed
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Why is the `// @ts-ignore` required here ? It is possible to specify the error that this line suppress ?
For example: 
```
  // @ts-ignore: Unreachable code error
```